catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/Gml3ToWktImpl.java

```
@@ -13,43 +13,55 @@
-import org.codice.ddf.parser.Parser;
-import org.codice.ddf.parser.ParserConfigurator;
-import org.codice.ddf.parser.ParserException;
+import java.util.function.Function;
+import javax.xml.parsers.ParserConfigurationException;
 import org.codice.ddf.transformer.xml.streaming.Gml3ToWkt;
-import org.jvnet.jaxb2_commons.locator.DefaultRootObjectLocator;
-import org.jvnet.ogc.gml.v_3_1_1.jts.ConversionFailedException;
-import org.jvnet.ogc.gml.v_3_1_1.jts.GML311ToJTSGeometryConverter;
+import org.geotools.geometry.jts.JTS;
+import org.geotools.referencing.CRS;
+import org.geotools.referencing.crs.DefaultGeographicCRS;
+import org.geotools.xml.Configuration;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.operation.MathTransform;
+import org.opengis.referencing.operation.TransformException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
 
 public class Gml3ToWktImpl implements Gml3ToWkt {
-  private final Parser parser;
+  private static final Logger LOGGER = LoggerFactory.getLogger(Gml3ToWkt.class);
 
-  private final ParserConfigurator configurator;
+  private final MathTransform latLonTransform;
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(Gml3ToWkt.class);
+  private final org.geotools.xml.Parser parser;
+
+  private final WKTWriter wktWriter = new WKTWriter();
 
-  public Gml3ToWktImpl(Parser parser) {
-    this.parser = parser;
-    this.configurator =
-        parser.configureParser(
-            Collections.singletonList(AbstractGeometryType.class.getPackage().getName()),
-            Gml3ToWktImpl.class.getClassLoader());
+  public Gml3ToWktImpl(Configuration gmlConfiguration) {
+    MathTransform transform = null;
+    try {
+      transform = CRS.findMathTransform(DefaultGeographicCRS.WGS84, CRS.decode("EPSG:4326", false));
+    } catch (FactoryException e) {
+      LOGGER.warn("Couldn't create lat/lon transform");
```

Looks like if we don't create this transform the whole class/service is no good. Wonder if we should be catching this at all.

---

If we should throw an exception, what would be appropriate?